docs/status.md

```diff
@@ -42,7 +42,7 @@ one of the following tracking labels.
 | atan2                    | no            | yes*         | yes*           | yes             | no          |
 | batch_norm_grad          | no            | yes*         | yes*           | no              | no          |
 | batch_norm_inference     | no            | yes*         | yes*           | no              | no          |
-| batch_norm_training      | no            | yes*         | yes*           | no              | no          |
+| batch_norm_training      | yes           | yes          | yes            | no              | no          |
```

Comment:
Similarly to https://github.com/openxla/stablehlo/pull/377#discussion_r1013445041, let's change verification to `revisit` while we're figuring out #369.